shared/src/lib.rs

```diff
@@ -499,7 +499,7 @@ pub struct ProcessingM2 {
     pub th_2: BytesHashLen,
     pub x: BytesP256ElemLen,
     pub g_y: BytesP256ElemLen,
-    pub plaintext_2: EdhocMessageBuffer,
+    pub plaintext_2: BufferPlaintext2,
```

Comment:
Is there a reason why some places are `plaintext: &[u8]` and others are `plaintext_2: BufferPlaintext2` ?